### PR TITLE
docs(examples): add multiple consumers usage

### PR DESCRIPTION
**Pull request by VBetsun**
It was not clear how to use  [multiple consumers](https://github.com/fastify/help/issues/600). So, that is why this example has been added

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ fastify.listen(3000, err => {
   console.log(`server listening on ${fastify.server.address().port}`)
 })
  ```
+ 
+for more examples please take a look at the [examples directory](./examples)
```
Comment (darkgl0w) [inline README.md:62]:
```suggestion
For more examples on how to use this plugin you can take a look at the [examples directory](./examples).
```

File: package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "infra:start": "docker-compose up -d",
     "infra:stop": "docker-compose stop",
-    "test": "standard && tsd && tap -j=1 test/*.js"
+    "lint": "standard --fix",
+    "test": "standard && tsd && tap test/*.js"
```
Comment (darkgl0w) [inline package.json:11]:
```suggestion
    "lint": "standard",
    "lint:fix": "standard --fix",
    "test": "npm run lint && tap \"test/*.js\" && npm run test:typescript",
    "test:typescript": "tsd"
```

File: examples/multiple-consumers.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+const crypto = require('crypto')
+const fastifyKafka = require('..')
+const fastify = require('fastify')
+
+const group = crypto.randomBytes(20).toString('hex')
+const logger = { level: 'debug' }
+const consumerOptions = {
+  consumer: {
+    'metadata.broker.list': '127.0.0.1:9092',
+    'group.id': group,
```
Comment (darkgl0w) [inline examples/multiple-consumers.js:12]:
```suggestion
const groupId = crypto.randomBytes(20).toString('hex')
const logger = { level: 'debug' }
const consumerOptions = {
  consumer: {
    'metadata.broker.list': '127.0.0.1:9092',
    'group.id': groupId,
```

**Review by darkgl0w — APPROVED** [review]
Thank you for contribution :)
It is always good to have an example that illustrates how to use a package ^^

LGTM

File: README.md (hunk 1 above)
Comment (VBetsun) [reply README.md]: changed

File: package.json (hunk 2 above)
Comment (VBetsun) [reply package.json]: updated

File: examples/multiple-consumers.js (hunk 3 above)
Comment (VBetsun) [reply examples/multiple-consumers.js]: renamed

**Review by Eomm — APPROVED** [review]

**Pull request merged**